docs/compute/drivers/cloudscale.rst

```diff
@@ -0,0 +1,75 @@
+Cloudscale Compute Driver Documentation
+=======================================
+
+`cloudscale.ch`_ is a public cloud provider based in Switzerland.
+
+.. figure:: /_static/images/provider_logos/cloudscale.png
+    :align: center
+    :width: 200
+    :target: http://www.cloudscale.ch
+
+
+How to get an API Key
+---------------------
+
+Simply visit `<https://control.cloudscale.ch/user/api-tokens>`_ and start
+having fun!
+
+You can generate read and read/write API keys. These token types give you more
+access control. Revoking an API token is also possible.
+
+Using the API to the full extent
+--------------------------------
+
+Most of the `cloudscale.ch` API is covered by the simple commands:
+
+- ``driver.list_sizes()``
+- ``driver.list_images()``
+- ``driver.list_sizes()``
+- ``driver.reboot_node(node)``
+- ``driver.ex_start_node(node)``
+- ``driver.ex_stop_node(node)``
+- ``driver.ex_node_by_uuid(server_uuid)``
+- ``driver.destroy_node(node)``
+- ``driver.create_node(name, size, image, ex_create_attr={})``
+
+In our :ref:`example <cloudscale-examples>` below you can see how you use 
+``ex_create_attr`` when creating servers. Possible dictionary entries in 
+``ex_create_attr`` are:
+
+- ``ssh_keys``: ``list`` of ``strings``; A list of SSH public keys.
+- ``volume_size_gb``:  ``integer``; The size in GB of the root volume.
+- ``bulk_volume_size_gb``: ``integer``; The size in GB of the bulk storage volume.
+- ``use_public_network``: ``boolean`` Attaching/Detaching the public network interface.
+- ``use_private_network``: ``boolean`` Attaching/Detaching the private network interface.
+- ``use_ipv6``: ``boolean`` Enabling/Disabling IPv6.
+- ``anti_affinity_with``: ``string``; Pass the UUID of another server.
+- ``user_data``: ``string``; Cloud-init configuration (cloud-config). Provide YAML.
```

Comment:
I'd use Python's standard names for the types here (i.e. `bool`, `str`, …).

Also, is there a standard convention for how to format such lists with type information, e.g. [here](http://libcloud.readthedocs.io/en/latest/compute/drivers/exoscale.html#libcloud.compute.drivers.exoscale.ExoscaleNodeDriver.create_node) the format `<id> (<type>) – <description>` is used.